WORKSPACE

```diff
@@ -220,6 +221,14 @@ pypi_repository(
     version = "3.4.0",
 )
 
+pypi_wheel_repository(
```

Comment:
Everything is working without the Tensorflow subrepository (I might need the pyserving part for further tests)